server/audit_math/bravo.py

```diff
@@ -342,6 +344,12 @@ def get_sample_size(
 
     samples: Dict = {}
 
+    if round_sizes:
+        num_sampled = sum([round_sizes.get(rnd, 0) for rnd in round_sizes])
```

Comment:
small nit: pretty sure it's impossible for `round_sizes.get(rnd, 0)` to hit the default case and give 0, since you're iterating over `round_sizes`. You could probably just say `sum(round_sizes.values())`. not a big deal though